artemis-server/src/test/java/org/apache/activemq/artemis/core/config/impl/FileConfigurationTest.java

```diff
@@ -64,6 +64,13 @@
 
 public class FileConfigurationTest extends ConfigurationImplTest {
 
+   static {
```

Comment:
Please, do not leave stuff after tests.

There's a rule on ActiveMQTestBae to cleanup properties, and that's probably why your test failed here.

So, you should use a BeforeClass and AFterClass to cleanup such thing.


I'm adding here and squashing the change as yours.

Reply:
I am adding this code:


```java
   @BeforeClass
   public static void setupProperties() {
      System.setProperty("a2Prop", "a2");
      System.setProperty("falseProp", "false");
      System.setProperty("trueProp", "true");
      System.setProperty("ninetyTwoProp", "92");
   }

   @AfterClass
   public static void clearProperties() {
      System.clearProperty("a2Prop");
      System.clearProperty("falseProp");
      System.clearProperty("trueProp");
      System.clearProperty("ninetyTwoProp");
   }

```

Reply:
@jbertram I added a checkstyle issue during my merge. My next PR will fix it